static/app/views/alerts/rules/issue/index.spec.jsx

```diff
@@ -151,6 +151,45 @@ describe('IssueRuleEditor', function () {
     ProjectsStore.reset();
   });
 
+  describe('Viewing the rule', () => {
+    const rule = TestStubs.MetricRule();
+    const permissionAlertText =
+      'These settings can only be edited by users with the organization-level owner, manager, or team-level admin roles.';
```

Comment:
nit: you can define this as a const in the component so if text is modified in the future test stays valid.